src/Components/Contacts/ContactItem.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const ContactItem = () => {
+    return (
```

Comment:
If your component return just dummy HTML and there is no methods or other functional except  HTML please use this format.

const ContactItem = () => ([YOUR_HTML])

Reply:
I just change this component on the next branches,any way i fix a few components with the same issue